lib/livebook_web/live/session_live/files_list_component.ex

```diff
@@ -168,4 +182,10 @@ defmodule LivebookWeb.SessionLive.FilesListComponent do
 
     {:noreply, socket}
   end
+
+  def handle_event("clear_file_entry_cache", %{"name" => name}, socket) do
+    file_entry = Enum.find(socket.assigns.file_entries, &(&1.name == name))
```

Comment:
Should we handle the case where nothing is found? Maybe two people click on the button at the same or I click twice too fast?

Reply:
Good call, updated for the other events too :)